tests/utils/test_helpers.py

```diff
@@ -76,3 +80,18 @@ def test_get_client_cert(config: "Config"):
     config.merge({"certificates": {"foo": {"client-cert": client_cert}}})
 
     assert get_client_cert(config, "foo") == Path(client_cert)
+
+
+def test_get_file_hash():
+
+    with tempfile.TemporaryDirectory() as tmp_dir:
+
+        test_data = [b"", b"12345", b"123" * 8000]
+
+        for i, data in enumerate(test_data):
+
+            tmp_file = Path(tmp_dir) / f"file{i}"
+            tmp_file.write_bytes(data)
+
+            real_hash = hashlib.md5(data).hexdigest()
+            assert get_file_hash(tmp_file, "md5") == real_hash
```

Comment:
The same as above, I don't think you need that many empty lines.

Reply:
fixed